network-api/networkapi/templates/donate/pages/base.html

```diff
@@ -30,7 +30,11 @@
 
 {% block bodyclass %} donate tw-m-0 tw-font-sans tw-font-normal tw-text-base tw-font-normal tw-text-black tw-bg-white tw-scroll-smooth {% endblock %}
 
-{% block donate_banner %}{% endblock %}
+{% block donate_banner %}
+    {% if settings.donate_banner.SiteDonateBanner.active_donate_banner %}
+        {% include "partials/donate_banner.html" with banner=settings.donate_banner.SiteDonateBanner.active_donate_banner.localized %}
```

Comment:
I'm not sure if we should use `.localized` here...  this is how we're doing it in the base, so I guess it's fine, but maybe we should create a ticket to move away from that...

Reply:
Thanks @jhonatan-lopes thats a good point especially after all the work we have done to remove .localized everywhere else in the code base. I have created https://github.com/MozillaFoundation/foundation.mozilla.org/issues/11404 to capture this work 👍